### PR TITLE
Allow new param, don't require old param.

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
TAG_REVIEWER will you please review this?

Passing `email` as the cardholder's email instead of the logged-in user's email breaks the logging for this request.

### Fixed Issues
https://github.com/Expensify/Expensify/issues/232931

# Tests
1. What unit/integration tests cover your change? What autoQA tests cover your change?
1. What tests did you perform that validates your changed worked?

# QA
1. What does QA need to do to validate your changes?
1. What areas to they need to test for regressions?

### CODE REVIEW

**Comment by tylerkaraszewski** [comment]
I have read the CLA Document and I hereby sign the CLA

File: lib/API.jsx [hunk 1]
```diff
@@ -762,10 +762,12 @@ export default function API(network, args) {
 
         card: {
             /**
-             * Set the limit of an expensify card
+             * Set the limit of an expensify card. We need to pass either `email` or `cardUserEmail`.
+             * `cardUserEmail` is correct, but email is still supported across the rollout to `cardUserEmail`.
```
Comment (flodnv) [inline lib/API.jsx:766]: Let's have a followup GH to clean this up after it's released?

**Review by flodnv — APPROVED** [review]

File: lib/API.jsx (hunk 1 above)
Comment (tylerkaraszewski) [reply lib/API.jsx:766]: Yes, I'll make that.

**Comment by tylerkaraszewski** [comment]
I'm going to merge this @dangrous but feel free to comment later if you have any concerns.

**Comment by dangrous** [comment]
Works for me! I guess this could now in theory have a usage that slips through the cracks with neither parameter, but that seems relatively unlikely?

**Pull request merged**